### ISSUE TITLE
Broken links in README.md

### ISSUE THREAD

**Issue by natros** [tdf/odftoolkit#85]
The links for documentation are  broken:

- The Home Page for the ODF Toolkit
- ODFDOM Getting Start Guide
- ODF Validator Getting Start Guide
- ODF XSLT Runner Getting Start Guide

**Comment by svanteschubert** [comment]
Thank you for the report and sorry that we were so distracted for a long time.
The earlier web generator "Apache CMS" was no longer maintained and showed unresolvable problems. There was no time to start the Sphinx project (I still would love to try this out, I am in love with the MongoDB documentation).
And Sebastian's work and my work on an intermediate replacement stopped somehow a year ago for many mixed reasons.
Long story short, we are about to fix this, see https://github.com/tdf/odftoolkit/issues/29#issuecomment-871975395
